test/broadcastable/test_helper_test.rb

```diff
@@ -1,269 +1,263 @@
 require "test_helper"
 
-module Turbo
```

Comment:
This is undoing the workaround I had to do in the original PR because it's no longer needed after this change. It should now be consistent with the other tests and what was in `main` prior to my original PR.